### DISCUSSION TITLE
TestConnection doesn't graphically display its result

### DISCUSSION THREAD

**Discussion by bgribaudo** [microsoft/vscode-powerquery-sdk#184] [Ideas]
Not sure if this a feature request, bug or something that is as intended. :-)

Most commands in the "Power Query SDK" sidebar display some kind of graphical output when run. For example, evaluating a file outputs various tabs to "PQTest result" while setting credentials displays a little "(i)" toast notification about credentials being set successfully.

However, "Run TestConnector function" is an exception to the above. Its output only shows up in the `Output` pane in raw, command-line style. Should it output its results graphically in a new pane, similar to "PQTest reuslt, or display a toast notification?

**Comment by ptyx507x** [comment]
hey! this is currently by-design, but I'm sending this to the discussions section so we can consider it an idea / feature request.

What type of notification or visual would you have in mind?

**Reply by bgribaudo** [reply]
Maybe it's as simple as a blue (i) toast notification saying "TestConnector test result: Success" (or "Failed").

Or it could result in opening a new "TestConnector result" tab that shows the request start and end time and something like "Result: Passed".

**Reply by mattmasson** [reply]
I like the idea of displaying a toast notification showing pass or fail. I believe we do something similar when setting credentials. 

/cc @albertli-msft
